packages/nimble-components/src/button/specs/README.md

```diff
@@ -0,0 +1,45 @@
+# Nimble-Button New Visual Designs Implementation
+
+## Problem Statement
+
+The Visual Design spec for Nimble buttons now includes primary button states `primary-mono` and `primary` that aren't implemented in the nimble-button component yet.
+
+## Links To Relevant Work Items and Reference Material
+
+[Nimble Issue #663: Adopt nimble-button visual designs for states primary, quiet, and loud](https://github.com/ni/nimble/issues/663)
+
+[Figma Spec](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-77205)
+
+## Implementation / Design
+
+[FAST Button API](https://github.com/microsoft/fast/blob/de7f234ef871204fcac2b5df59433d919809341d/packages/web-components/fast-foundation/src/button/README.md)
+
+The appearance of the button will have states `default` (White), `primary-mono` (Black), and `primary` (Green), each with its own color scheme based on the UI theme.
+`primary-mono` and `primary` states of buttons are primary buttons, so there should only be one of either in a section.
+
+The `primary-mono` button can be used when there is a conflict with color and its context.
+The `primary` button can be used in situations where a button needs to have the most prominent eye-catching approach, or when there is a lack of color.
+
+Ghost buttons will not have `primary-mono` or `primary` states.
+Outline and Block buttons in the Color UI will not have a `primary` state, as this has been declared as not recommended for this type of background.
```

Comment:
Currently ghost setting an `appearance-variant` on ghost buttons has no effect. I assume that will stay the same?

What about the two primary states in the Color theme? Would one apply a primary style and one would keep the component as though `appearance-variant` isn't set? Or, would both `appearance-variant` values just be rendered identically?

Reply:
Yes, the ghost setting will stay the same, although we could give its radio a functionality I'll mention in the final paragraph.

If I understand correctly, I think that if the user was to, for example, click `primary` on the radio button for the color theme, the button should have the default style, as though `appearance-variant` isn't set. If we instead have the values rendered identically it may be confusing.

Although, I feel that giving the option to select the value `primary` when it doesn't do anything could be confusing on its own. Would it be possible to have the radio button for `primary` disabled when the color theme is being used? Similarly, could the radio buttons `primary` and `primary-monochrome` be disabled when the ghost button is selected on any theme?

Reply:
Actually, I see that you already have outlined a proposal for this behavior in the spec.

I don't have an issue with the current proposal of styling a `primary-accent` button in the Color theme as though `appearance-variant` wasn't set on it at all.